shared/tomcat/bin/setenv.bat

```diff
@@ -0,0 +1 @@
+set "CLASSPATH=%CLASSPATH%;%CATALINA_BASE%\lib\azure.appservice.jar"
```

Comment:
>set [](start = 0, length = 3)

why you need this file?

Reply:
any changes to Jul needs to be loaded at system level. [https://tomcat.apache.org/tomcat-9.0-doc/class-loader-howto.html](https://tomcat.apache.org/tomcat-9.0-doc/class-loader-howto.html)

---
In reply to: [263233479](https://github.com/Azure-App-Service/tomcat/pull/22#discussion_r263233479) [](ancestors = 263233479)

Reply:
Can you add this info as a comment?